standup/auth0/middleware.py

```diff
@@ -0,0 +1,121 @@
+from datetime import datetime, timedelta
```

Comment:
None of this is tested or hooked up to anything in this PR. So the code in this file is just hanging out. Feel free to skip it.